avocado/utils/cpu.py

```diff
@@ -51,6 +51,20 @@ def _get_cpu_info():
     return cpuinfo
 
 
+def _get_cpu_status(cpu):
+    """
+    Check if a CPU is online or offline
+
+    :pram cpu: CPU number 1 or 2 or 39
+    :type cpu: string
```

Comment:
nitpick: It could be int, string or whatever which provides `__str__` 😉. I'd go with `int` but this is also understandable.